src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/SourceItemHandler.cs

```diff
@@ -121,6 +121,8 @@ public override Task OnContextReleasedAsync(IWorkspaceProjectContext context)
 
         private void RemoveSourceFile(string fullPath, IWorkspaceProjectContext context)
         {
+            fullPath = _project.MakeRooted(fullPath);
```

Comment:
Can we please unify this? We're already making rooted in the evaluation case. Can you remove those cases?